sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py

```diff
@@ -51,6 +51,54 @@ def __init__(self, codes):
             ),
 
             CodeRegField(consts.CONNECTOR_FIELD, self.getaddr(0x0, 203), self.codes.CONNECTORS),
+
```

Comment:
In the case of Python2, these will cause Python errors in the pytest.
Please be sure to use **python3 -m pytest -sx**